rs/rosetta-api/icp/client/src/lib.rs

```diff
@@ -205,6 +208,36 @@ impl RosettaClient {
         }])
     }
 
+    pub async fn build_set_dissolve_timestamp_operations(
+        signer_principal: Principal,
+        neuron_index: u64,
+        // The number of seconds since Unix epoch.
+        // The dissolve delay will be set to this value
+        // The timestamp has to be in the future and greater or equal to the currently set timestamp
+        timestamp: u64,
+    ) -> anyhow::Result<Vec<Operation>> {
+        Ok(vec![Operation {
+            operation_identifier: OperationIdentifier {
+                index: 0,
+                network_index: None,
+            },
+            related_operations: None,
+            type_: "SET_DISSOLVE_TIMESTAMP".to_string(),
+            status: None,
+            account: Some(AccountIdentifier::new(PrincipalId(signer_principal), None).into()),
```

Comment:
I find the `.into()` sub-optimal here, but the conversion is from an `icp_ledger::account_identifier::AccountIdentifier` to a `rosetta_core::identifiers::AccountIdentifier`, so perhaps this is still an acceptable option.